src/main/java/org/junit/experimental/categories/Categories.java

```diff
@@ -346,11 +347,28 @@ private static boolean hasAssignableTo(Set<Class<?>> assigns, Class<?> to) {
         return false;
     }
 
-    private static Set<Class<?>> createSet(Class<?>... t) {
-        final Set<Class<?>> set= new HashSet<Class<?>>();
-        if (t != null) {
-            Collections.addAll(set, t);
+    private static Set<Class<?>> createSet(Class<?>[] t) {
```

Comment:
While you're at it, can you please rename `t` to something with a little more meaning?